generate-docker-env.sh

```diff
@@ -15,7 +15,7 @@ if [ -f .env ]; then
 fi
 
 # get current branch
-BRANCH=$(git rev-parse --abbrev-ref HEAD)
+BRANCH=$(git rev-parse --abbrev-ref HEAD 2>/dev/null || echo main)
```

Comment:
Asume `main` (which ultimately means the published `ghcr.io` images) if `generate-docker-env.sh` is run from outside of a git clone.